iphone/Classes/TiAppiOSUserActivityProxy.m

```diff
@@ -175,11 +184,6 @@ -(NSNumber*)isValid
     return NUMBOOL(_isValid);
 }
 
--(NSNumber*)supported
```

Comment:
We should not remove the existing ones, since they are already used by developers.